test/libsolidity/syntaxTests/modifiers/access_in_library.sol

```diff
@@ -0,0 +1,9 @@
+library L {
+    modifier m() { _; }
+}
+contract C {
+    function f() L.m public {
+    }
+}
+// ----
+// TypeError 9428: (68-71): Can only use modifiers defined in the current contract or in base contracts.
```

Comment:
Does `using` declaration permit access to modifiers defined in libraries?

```
library L {
    modifier m() { _; }
}
contract C {
    using L for *;
    function f() L.m public {
    }
}
```

Reply:
i don't think so - using does not work like that, but I can add such a test.